src/start/LoadingScreen.js

```diff
@@ -18,6 +18,7 @@ export default class LoadingScreen extends PureComponent<{||}> {
   render() {
     return (
       <View style={styles.center}>
+        <StatusBar backgroundColor={BRAND_COLOR} barStyle="light-content" />
```

Comment:
`backgroundColor` is an android-only prop. Have you tested how `barStyle="light-content"` looks in an IOS device, since `backgroundColor` is no-op for IOS?

Reply:
I don't have access to iOS devices, but a quick check in snacks.expo.io showed the icons in status bar aren't visible on white background in iOS, changing ``barStyle="light-content"`` to ``barStyle="dark-content"`` the icons remain black and visible, will fix this now.